pkg/agent/agent.go

```diff
@@ -2,6 +2,8 @@ package agent
 
 import (
 	"context"
+	update_api "github.com/bottlerocket-os/bottlerocket-update-operator/pkg/platform/update-api"
```

Comment:
This will come in as this import name that's declared in the `package` line: so this will `package update_api` without having to be `update_api "my/path/update-api"`.
```suggestion
	"github.com/bottlerocket-os/bottlerocket-update-operator/pkg/platform/update-api"
```